be/extmap.h

```diff
@@ -199,10 +199,10 @@ struct m0_be_emap_seg {
 struct m0_be_emap_cursor {
 	/** Map this cursor is iterating through. */
 	struct m0_be_emap        *ec_map;
-	/** Segment currently reached. */
-	struct m0_be_emap_seg     ec_seg;
 	/** Emap current version. */
 	uint64_t                  ec_version;
+	/** Segment currently reached. */
+	struct m0_be_emap_seg     ec_seg;
```

Comment:
What is the reason for this reorder here ?

Reply:
To compact the structure size.